torch/package/importer.py

```diff
@@ -35,7 +45,7 @@ class PackageImporter:
     """The dictionary of already loaded modules from this package, equivalent to `sys.modules` but
     local to this importer.
     """
-    modules : Dict[str, Optional[types.ModuleType]]
+    modules : Dict[str, ModuleType]
```

Comment:
This is wrong. `None` gets inserted if a module fails to load. Or it is allowed if you want to force a module to fail to load.

Reply:
I looked a bit and could not find the place where we insert None, but possible I don't understand importlib. Can you point me to it?